client_libraries-DEPRECATED/python/src/example_client.py

```diff
@@ -222,44 +213,52 @@ def main():
 
 
 def get_survey(cs, survey_id):
-    return cs.surveys().get(surveyUrlId=survey_id).execute()
+    """Gets a survey.
+
+    Args:
+        cs: The Surveys Service used to send the HTTP requests.
+        survey_id: The id of the survey to get.
+
+    Returns:
+        A dictionary containing the survey fields.
+    """
+    result = cs.surveys().get(surveyUrlId=survey_id).execute()
+    return result
 
 
 def list_surveys(cs):
     """Prints the surveys that are owned by the given user.
 
     Args:
-        cs: The Consumer Surveys Service used to send the HTTP requests.
-
-    Returns:
-        A dictionary containing the survey id of the started survey.
+        cs: The Surveys Service used to send the HTTP requests.
     """
     results = cs.surveys().list().execute()
-    for s in results.resources:
-      print '%s' % s.surveyUrlId
+    for s in results.get('resources'):
+        print "ID: %s, Title: %s" % (s.get('surveyUrlId'), s.get('title'))
```

Comment:
Updated to pretty print the survey list results (along with the get() survey results).